payment/src/main/java/org/killbill/billing/payment/invoice/InvoicePaymentControlPluginApi.java

```diff
@@ -179,9 +180,8 @@ public OnSuccessPaymentControlResult onSuccessCall(final PaymentControlContext p
                             invoicePaymentAmount = paymentControlContext.getAmount();
                         }
 
-                        log.debug("Notifying invoice of {} paymentId='{}', amount='{}', currency='{}', invoiceId='{}'", success ? "successful" : "pending", paymentControlContext.getPaymentId(), invoicePaymentAmount, paymentControlContext.getCurrency(), invoiceId);
+                        log.debug("Notifying invoice of {} paymentId='{}', amount='{}', currency='{}', invoiceId='{}' invoicePaymentStatus='{}'", status, paymentControlContext.getPaymentId(), invoicePaymentAmount, paymentControlContext.getCurrency(), invoiceId);
```

Comment:
⚠️ Wrong ordering / numbering of arguments.

Reply:
Fixed in https://github.com/killbill/killbill/pull/1802/commits/580bee255fc583ae6ca799ff622cc1beede1ac75